src/components/label/Label.tsx

```diff
@@ -30,9 +30,20 @@ export interface LabelProps {
     text: React.ReactNode;
     /** Optional tooltip text for the label */
     tooltip?: React.ReactNode;
+    /** Optional peopierties for label element */
+    labelElProps?: Object;
```

Comment:
```suggestion
    /** Optional props for the label element */
    labelElProps?: Object;
```

same for the other props list